resources/kubernetes/operator/helm/values.yaml

```diff
@@ -180,19 +213,31 @@ collectors:
             exporters:
             - debug
             - elasticsearch/otel
+
+  #  Daemon is a K8s daemonset EDOT collector focused on gathering telemetry at
+  #  node level and expose an OTLP endpoint for data ingestion.
```

Comment:
exposing

Reply:
> OTLP endpoint for data ingestion.

not sure about the below (?)
OTLP endpoint for data ingestion. SDKs using auto-instrumentation will use this